Library/Formula/xdgdirs.rb

```diff
@@ -0,0 +1,47 @@
+class Xdgdirs < Formula
+  desc "Display the XDG_* env vars related to the XDG Basedir Specification"
+  homepage "http://www.gnuvola.org/software/xdgdirs/"
+  url "http://www.gnuvola.org/software/xdgdirs/xdgdirs-2.2.tar.xz"
+  sha256 "a7e3543eee10758a8ae6e493900c44ab946153887d56a46b96ae67f4d589e68c"
+
+  depends_on "guile"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    require "json"
+    require "open3"
+
+    # Clear out the environment of any existing XDG env vars to get consistent
+    # behavior for the test
+    new_env = ENV.reject { |name, _value| name.start_with? "XDG_" }
+
+    xdg_str, stderr, status = Open3.capture3(new_env, "#{bin}/xdgdirs -l json homebrew")
+    xdg_values = JSON.parse(xdg_str)
+
+    assert_equal 0, status, "xdgdirs failed, stderr:\n#{stderr}"
+    assert_equal "#{Dir.home}/.config/homebrew", xdg_values["config-home"]
+    assert_equal "#{Dir.home}/.local/share/homebrew", xdg_values["data-home"]
+    assert_equal "#{Dir.home}/.cache/homebrew", xdg_values["cache-home"]
+    assert_equal ["/usr/local/share/homebrew", "/usr/share/homebrew"], xdg_values["data-dirs"]
```

Comment:
I just used the string `"homebrew"` for the test, it could literally be anything. I was just making sure that absent any `XDG_` env vars, the program would give the correct default locations for the various paths.